colab_notebook_utils/datatypes.py

```diff
@@ -1,4 +1,82 @@
+from typing import Optional
 from enum import Enum
+from dataclasses import dataclass
+
+
+class MocapType(str, Enum):
+    HAWKEYE = "hawkeye"
+    HAWKEYE_HFR = "hawkeyehfr"
+
+
+class MovementType(str, Enum):
+    BASEBALL_PITCHING = "baseball-pitching"
+    BASEBALL_HITTING = "baseball-hitting"
+
+
+class Handedness(str, Enum):
+    RIGHT = "right"
+    LEFT = "left"
+
+    @property
+    def eye_hand_multiplier(self) -> int:
+        return 1 if self == Handedness.RIGHT else -1
+
+
+class FileType(str, Enum):
+    INVERSE_KINEMATICS = "inverse-kinematics"
+    IK_AT_TIME_POINTS = "ik-at-time-points"
+    METRICS_BASEBALL_PITCHING_V1 = "metrics-baseball-pitching-v1-0-0"
+    METRICS_BASEBALL_PITCHING_V2 = "metrics-baseball-pitching-v2-0-0"
+    METRICS_BASEBALL_HITTING_V1 = "metrics-baseball-hitting-v1-0-0"
+    METRICS_BASEBALL_HITTING_V2 = "metrics-baseball-hitting-v2-0-0"
+
+
+@dataclass
+class S3Metadata:
+    org_id: str
+    mocap_types: list[MocapType]
+    movement_type: MovementType
+    handedness: Handedness
+    file_type: FileType
+
+    @property
+    def s3_population_prefix(self) -> str:
+        if len(self.mocap_types) > 1:
+            print('Input multiple mocap_types, setting the mocap type as the first item in the list')
+
+        mocap_type = self.mocap_types[0]
+
+        return f"s3://reboot-motion-{self.org_id}/population/{mocap_type}/{self.movement_type}/" \
+               f"{self.file_type}/000000_{self.movement_type}_{self.handedness}_"
+
+
+@dataclass
+class PlayerMetadata:
+    org_player_ids: Optional[list[str]] = None

```

Comment:
if you want to default this to an empty list, you can do 
`org_player_ids: List[str] = field(default_factory=list)`

Reply:
ah ok cool, for now I actually have None being used explicitly in other areas to denote nothing was entered, but in the future we could use an empty list